packages/core/src/lib/waku.ts

```diff
@@ -51,25 +51,17 @@ export class WakuNode implements Waku {
   public filter?: IFilter;
   public lightPush?: ILightPush;
   public connectionManager: ConnectionManager;
-  public readonly pubsubTopics: PubsubTopic[];
 
   constructor(
     options: WakuOptions,
-    pubsubTopics: PubsubTopic[] = [],
+    public readonly pubsubTopics: PubsubTopic[],
     libp2p: Libp2p,
-    private pubsubShardInfo?: ShardingParams,
+    private pubsubShardInfo?: ShardInfo,
```

Comment:
we can remove `pubsubShardInfo` and the getter for `shardInfo` can be removed as well
if not remove, we can add the conversion of `pubsubTopic` to `shardInfo` within the getter since that's the only place it seems to be used now

this way only `PubsubTopic[]` is an argument for the constructor

Reply:
removed `ShardInfo` in latest commit